easybuild/easyconfigs/c/cwltool/cwltool-3.1.20221018083734-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,117 @@
+##
+# This is a contribution from SIB Swiss Institute of Bioinformatics
+# Homepage:     https://www.sib.swiss/research-infrastructure/competence-centers/vital-it
+#
+# Authors::     Sebastien Moretti <sebastien.moretti@sib.swiss>
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'cwltool'
+version = '3.1.20221018083734'
+
+homepage = 'https://github.com/common-workflow-language/cwltool'
+description = """Common workflow language (CWL) reference implementation."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake',    '3.20.1'),
+    ('make',     '4.3'),
+    ('Check',    '0.15.2'),
+    ('SWIG',     '4.0.2'),
+    ('expat',    '2.2.9'),
+    ('bzip2',    '1.0.8'),
+    ('zlib',     '1.2.11'),
+]
+
+# The Qt5 dependency of Graphviz (required by pydot) may be long to build and
+# requires a lot of disk space and memory!
+dependencies = [
+    ('Python',            '3.9.5'),
+    ('typing-extensions', '3.10.0.0'),
+    ('pydot',             '1.4.2'),
+    ('lxml',              '4.6.3'),
+]
+
+use_pip = True
+
+# Fetch the tar.gz, not the whl files!
+exts_list = [
+    ('argcomplete', '2.0.0', {
+        'checksums': ['6372ad78c89d662035101418ae253668445b391755cfe94ea52f1b9d22425b20'],
+    }),
+    ('humanfriendly', '10.0', {
+        'checksums': ['6b0b831ce8f15f7300721aa49829fc4e83921a9a301cc7f606be6686a2288ddc'],
+    }),
+    ('coloredlogs', '15.0.1', {
+        'checksums': ['7c991aa71a4577af2f82600d8f8f3a89f936baeaf9b50a9c197da014e5bf16b0'],
+    }),
+    ('ruamel.yaml.clib', '0.2.7', {
+        'modulename': 'ruamel.yaml',
+        'checksums': ['1f08fd5a2bea9c4180db71678e850b995d2a5f4537be0e94557668cf0f5f9497'],
+    }),
+    ('ruamel.yaml', '0.17.14', {
```

Comment:
Can we use the existing `ruamel.yaml-0.17.21-GCCcore-10.3.0.eb` as a dependency instead?

Why the older version?

Reply:
I missed than one. I will change that. ruamel.yaml < 0.17.22 is required so should work